web/src/main/webapp/xslt/ui-search/search-nojs.xsl

```diff
@@ -200,26 +205,22 @@
                          </xsl:otherwise>
                        </xsl:choose>
                      </a>
-                   </h3>
+                   </h1>
                  </div>
 
-                 <div>
-                   <div class="gn-md-thumbnail">
-                     <xsl:for-each select="image[1]">
-                       <img class="gn-img-thumbnail" itemprop="thumbnailUrl"
-                            src="{tokenize(., '\|')[2]}"></img>
-                     </xsl:for-each>
-                   </div>
-                   <div style="float:left; display:block; width: calc(100% - 162px)">
-                     <div class="text-justify gn-md-abstract ellipsis">
-                       <div>
-                         <p itemprop="description">
-                           <xsl:value-of select="abstract"/>
-                         </p>
-                       </div>
-                     </div>
-                   </div>
-                 </div>
+                  <div class="clearfix">
+                    <div class="gn-md-thumbnail pull-left">
```

Comment:
Not sure why but thumbnail and text may overlap when no thumbnail ?

![image](https://user-images.githubusercontent.com/1701393/57120932-bc0b4580-6d74-11e9-897a-7ea500a2e1e3.png)

The grey background is when you play with CSS settings.

Reply:
That doesn't look nice, I will have a look